### ISSUE TITLE
Wrong version in the CMakeLists.txt

### ISSUE THREAD

**Issue by GiulioRomualdi** [artivis/manif#232]
Hi @artivis,  first of all, thank you for releasing version `0.0.4`. 

I've just noticed that even if the latest version is `0.0.4`. However, the  `cmake` project version is still `0.0.3`. 

https://github.com/artivis/manif/blob/0d42c2de1a34628ca9044b1f2e4dd17bbfead904/CMakeLists.txt#L4

**Comment by GiulioRomualdi** [comment]
The `package.xml` file contains also the wrong version  

https://github.com/artivis/manif/blob/0d42c2de1a34628ca9044b1f2e4dd17bbfead904/package.xml#L4

**Comment by artivis** [comment]
Hi,

indeed the tagging is partially messed up, something went wrong with the script I'm using.
If you `git checkout 0.0.4`, the `package.xml` has the correct version but not the `CMakeLists.txt`. However they both have the proper version on the `master` branch...
Gonna fix that manually.

**Comment by artivis** [comment]
This is now fixed on tag 0.0.4:

```bash
$ git clone https://github.com/artivis/manif -b 0.0.4 && cd manif
$ head package.xml | grep '<version>'
  <version>0.0.4</version>
$ head CMakeLists.txt | grep 'manif VERSION'
project(manif VERSION 0.0.4 LANGUAGES CXX)
```
